debian/changelog

```diff
@@ -1,3 +1,27 @@
+opx-nas-linux (5.25.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
```

Comment:
Need more information

Reply:
@rakeshdatta  please clarify what you need more information about.

Reply:
This should say `* Feature: Static L2 VxLAN`

Reply:
fixed